### DISCUSSION TITLE
Preprocessing of diffusion MRI

### DISCUSSION THREAD

**Discussion by Sunita2023** [dipy/dipy#3192] [Preprocessing]
Dear DIPY users,

How to preprocess diffusion MRI DICOM images? So far we have been using publicly available processed DWI nifti file and the given .bvec and  .bval files. Now, we would like to do preprocessing from DICOM file. However, there are number of questions. 

We used dcm2niix library to convert DICOM image into nifti file as the first step. We checked both DTI and DWI images. One of the DTI files was showing 4 dimension while DWI images were all 3 dimensional. During the processing, it generated nifti file along with .bval and .bvec files. However, the .bval and .bvec files were containing only zero instead of gradient directions. Are we looking at the correct dMRI file? We are still in confusion whether we have to look for DTI or DWI file.  

We also looked for the tags (0018, 9087) Diffusion b-value, (0018, 9089) Diffusion Gradient Orientation where all values were 0 in DTI file. Could you please guide us which is the appropriate DICOM file to look into which can provide us the required .bvec and .bval values. Is there any other tag we should look for to generate these files?

Beside what are the other important steps to complete the preprocessing before we start generating the fiber tracts?

I would really appreciate DIPY forum members for been so helpful.  

Thanks you so much.

Best Regards,
Sunita

**Comment by arokem** [comment]
Hi Sunita, 

One case in which I have seen similar - b-vectors all set to zero upon conversion from dicom to nifti - is when an anonymization algorithm applied when exporting the data from a clinical PACS system did not recognize these fields as crucial metadata, and assumed that this is information that should be "anonymized" too. Unfortunately, without this information, you will not be able to usefully process the diffusion data, so you will have to go back to the source of the data and see whether the anonymization algorithm can be adjusted to let the information in these fields through. Or it may be something else...

**Reply by Sunita2023** [reply]
Thanks a lot Ariel. I am sure this could be the reason. We will check the source.
